src/components/RegistrationForm.tsx

```diff
@@ -64,11 +70,13 @@ export default function RegistrationForm({ handles, wardens, className }) {
   const [status, setStatus] = useState<FormStatus>(FormStatus.Unsubmitted);
   const [errorMessage, setErrorMessage] = useState<string | ReactNode>("");
   const [isDangerousUsername, setisDangerousUsername] = useState(false);
+  const [hasValidateLinkError, setHasValidateLinkError] = useState(false);
   const [captchaToken, setCaptchaToken] = useState("");
 
   // global variables
   const avatarInputRef = useRef<HTMLInputElement>();
   const discordUsernameRegex = new RegExp(/.*#[0-9]{4}/, "g");
+  const checkAbsoluteURL = new RegExp("^(?:[a-z]+:)?//", "i");
```

Comment:
Specifically http(s)?

```suggestion
  const checkAbsoluteURL = new RegExp("^(http[s]?)://", "i");
```